ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_SERVERS.yml

```diff
@@ -389,6 +389,127 @@ servers:
           mode: active
           enabled: false
 
+  server18_monitoring_session_source_phys_interfaces:
+    rack: RackC
+    adapters:
+      - endpoint_ports: [ Eth1, Eth2 ]
+        switch_ports: [ Ethernet25, Ethernet25 ]
+        switches: [ DC1-SVC3A, DC1-SVC3B ]
+        profile: TENANT_A
+        monitor_sessions:
+          - name: MonitoringSessionServer18WithDest
+            role: source
+            source_settings:
+              direction: rx
+              access_group:
+                type: ip
+                name: MyIpACL
+                priority: 5
+          - name: MonitoringSessionServer18WithoutDest
+            role: source
+            source_settings:
+              direction: tx
+
+  server18_monitoring_session_source_po:
+    rack: RackC
+    adapters:
+      - server_ports: [ Eth3, Eth4 ]
+        switch_ports: [ Ethernet27, Ethernet27 ]
+        switches: [ DC1-SVC3A, DC1-SVC3B ]
+        profile: TENANT_A
+        enabled: true
+        port_channel:
+          description: server18_monitoring_session_source_po
+          mode: active
+          enabled: true
+        monitor_sessions:
+          - name: MonitoringSessionServer18WithDest
+            role: source
+            source_settings:
+              direction: tx
+              access_group:
+                type: mac
+                name: MyMacACL
+                priority: 5
+
+  server18_monitoring_session_source_phys_interface:
+    rack: RackC
+    adapters:
+      - server_ports: [ Eth5 ]
+        switch_ports: [ Ethernet28 ]
+        switches: [ DC1-SVC3A ]
+        profile: TENANT_A
+        enabled: true
+        monitor_sessions:
+          - name: MonitoringSessionServer18WithDest
+            role: source
+            source_settings:
+              direction: tx
+              access_group:
+                type: mac
+                name: MyMacACL
+                priority: 5
+
+  server19_monitoring_session_destination_phys:
+    rack: RackC
+    adapters:
+      - endpoint_ports: [ Eth1, Eth2 ]
+        switch_ports: [ Ethernet26, Ethernet26 ]
+        switches: [ DC1-SVC3A, DC1-SVC3B ]
+        monitor_sessions:
+          - name: MonitoringSessionServer18WithDest
+            role: destination
+            session_settings:
+              encapsulation_gre_metadata_tx: true
+              header_remove_size: 20
```

Comment:
Please clarify this for me. Here **size = 20** for DC1-SVC3A and in line 486 it is **200**. Which value should it be taking? Similarly for other parameters.

Reply:
like: ip_acl and mac_acl

Reply:
Hi Tony, 
this is expected behaviour. 
"session_settings" is valid per session, so in this case the session is named "MonitoringSessionServer18WithDest".

First we have specified:

```
server19_monitoring_session_destination_phys:
    rack: RackC
    adapters:
      - endpoint_ports: [ Eth1, Eth2 ]
        switch_ports: [ Ethernet26, Ethernet26 ]
        switches: [ DC1-SVC3A, DC1-SVC3B ]
        monitor_sessions:
          - name: MonitoringSessionServer18WithDest
            role: destination
            session_settings:
              encapsulation_gre_metadata_tx: true
              header_remove_size: 20
              access_group:
                type: ip
                name: ip_acl
              rate_limit_per_ingress_chip: 30 bps
              rate_limit_per_egress_chip: 30 bps
              sample: 10
              truncate:
                enabled: true
                size: 20
```

for both switches and later

```
  server20_monitoring_session_destination_phys:
    rack: RackC
    adapters:
      - endpoint_ports: [ Eth1 ]
        switch_ports: [ Ethernet40 ]
        switches: [ DC1-SVC3A ]
        monitor_sessions:
          - name: MonitoringSessionServer18WithDest
            role: destination
            session_settings:
              encapsulation_gre_metadata_tx: false
              header_remove_size: 200
              access_group:
                type: mac
                name: mac_acl
              rate_limit_per_ingress_chip: 30 bps
              rate_limit_per_egress_chip: 30 bps
              sample: 10
              truncate:
                enabled: true
                size: 20
```

just for the session on: DC1-SVC3A

these settings will be combined/overwrite the previous ones, but only for DC1-SVC3A

hope this explains it?